packages/shared/src/styles/themes.scss

```diff
@@ -15,7 +15,7 @@
     --text-lh-l: 1.75;
     --text-lh-m: 1.5;
     --text-lh-s: 1.25;
-    --text-lh-x: 1.4;
+    --text-lh-xs: 1.4;
```

Comment:
If you refactor this, you need to ensure to update it everywhere this is used.

Reply:
You're right, this is now done.